app/views/users/new.html.erb

```diff
@@ -1,12 +1,27 @@
+<!-- <form action="/users" method="post">
+    First name: <input type="text" name="username"><br>
+    Email: <input type="text" name="email"><br>
+    password: <input type="text" name="password"><br>
+    <input type="hidden" name="authenticity_token" value="<%= form_authenticity_token %>">
+    <input type="submit" value="Submit">
+</form> -->
+
+<!-- <%= form_tag users_path do |form| %>
+    <%= label_tag "Usernamer" %>
+    <%= text_field_tag :username %><br>
+    <%= label_tag"Email" %>
+    <%= text_field_tag :email %><br>
+    <%= label_tag "Password" %>
+    <%= text_field_tag :password %><br>
+    <%= submit_tag "add user"  %>
+<% end %> -->
+
 <%= form_with model: @user do |u| %>
 <%= u.label :username, "Usernamer" %>
```

Comment:
i excluded users#edit because i don't have anything to pass to the edit template, given that i already passed the @user instance by calling the set_user method